dm/syncer/data_validator.go

```diff
@@ -199,7 +204,7 @@ type DataValidator struct {
 
 	// fields in this field block are guarded by stateMutex
```

Comment:
seems we can update the comment about which members this mutex guards

Reply:
which field do you mean? the range is not changed